test/conftest.py

```diff
@@ -193,6 +193,46 @@ def pudl_out_orig(live_dbs, pudl_engine):
     return PudlTabl(pudl_engine=pudl_engine)
 
 
+@pytest.fixture(scope="session")
+def ferc_to_sqlite_dbf_only(live_dbs, pudl_datastore_config, etl_settings, pudl_env):
```

Comment:
Would it be easy & make sense to further break these down by both data format and individual form? (which would require breaking down the underlying jobs as well)